jobs/resource_job.go

```diff
@@ -895,7 +895,15 @@ func ResourceJob() *schema.Resource {
 		},
 		Read: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
 			ctx = getReadCtx(ctx, d)
-			job, err := NewJobsAPI(ctx, c).Read(d.Id())
+			w, err := c.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			jobID, err := parseJobId(d.Id())
+			if err != nil {
+				return err
+			}
+			job, err := w.Jobs.GetByJobId(ctx, jobID)
 			if err != nil {
 				return err
 			}
```

Comment:
In line 911 we do 
```
return common.StructToData(*job.Settings, jobSchema, d)
```
job.Settings is now a different struct, with different tf tags. It may be lack of tests, but I am a bit surprised that this change works this easily.

Reply:
I would recommend to not to start with jobs until we figure out how to solve a problem with aliases

Reply:
I don't have context on the aliases and the problems. Can you share a reference?

Reply:
The job resource heavily uses `tf:"alias"` to rename fields, for example, in `git` block, we rename `git_url` into `url`, etc.